index.js

```diff
@@ -70,6 +73,11 @@ function createServer(argv) {
     var ldp = ldpApp.locals.ldp;
     app.use(ldp.mount, ldpApp);
 
+    //Error handling
+    app.use(errorHandler.handler);
```

Comment:
Yeah, that was the issue. I got the error handler to work just on the router and not in the whole app.